custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -78,11 +84,11 @@ def aggregation_query(self):
             ('cases_person_adolescent_girls_15_18_all',),
             ('daily_attendance_open', '0'),
             ('num_awcs',),
-            ('num_launched_states', '0'),
-            ('num_launched_districts', '0'),
-            ('num_launched_blocks', '0'),
-            ('num_launched_supervisors', '0'),
-            ('num_launched_awcs', '0'),
+            ('num_launched_states', ) if self.use_agg_awc else ('num_launched_states', '0'),
+            ('num_launched_districts',) if self.use_agg_awc else ('num_launched_districts', '0'),
+            ('num_launched_blocks',) if self.use_agg_awc else ('num_launched_blocks', '0'),
+            ('num_launched_supervisors',) if self.use_agg_awc else ('num_launched_supervisors', '0'),
+            ('num_launched_awcs',) if self.use_agg_awc else ('num_launched_awcs', '0'),
```

Comment:
nit: could consider pulling this out into a well-named function which would 

1. make modifying the logic easier in the future
2. serve as documentation for why this conditional exists

Reply:
in addition to what cory wrote the condition should be in the second element of the tuple not the entire tuple, there is no reason to replicate the column name